tests/vision/dataset_readers/flickr30k_test.py

```diff
@@ -8,11 +8,13 @@
 from allennlp.modules.vision.grid_embedder import NullGridEmbedder
 from allennlp.modules.vision.region_detector import RandomRegionDetector
 
+import random
 from tests import FIXTURES_ROOT
 
 
 class TestFlickr30kReader(AllenNlpTestCase):
     def setup_method(self):
+        random.seed(12345)
```

Comment:
What's the deal with this? Why does that make the tests less random?

Reply:
It doesn't, I was testing something but it didn't work. The test is passing locally, so it's hard to debug what's going wrong.